packages/fluentui/react-northstar-emotion-renderer/src/invokeKeyframes.ts

```diff
@@ -38,8 +38,10 @@ export function invokeKeyframes(cache: EmotionCache, styles: ICSSInJSStyle) {
 
         if (style.keyframe) {
           styles[property] = keyframes(cache, style.keyframe, style.params);
+          continue;
         }
 
+        styles[property] = keyframes(cache, style as AnimationKeyFrame, style.params);
```

Comment:
Can we add a test case for this in `packages/fluentui/react-northstar-emotion-renderer/test/emotionRenderer-test.ts`?